packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts

```diff
@@ -118,7 +132,120 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
      * exclusively by another plugin.
      * @param event The event to handle:
      */
-    onPluginEvent(_event: PluginEvent) {}
+    onPluginEvent(event: PluginEvent) {
+        if (!this.editor) {
+            return;
+        }
+        switch (event.eventType) {
+            case 'mouseUp':
+                this.mouseUpHandler(this.editor, event);
+                break;
+            case 'keyDown':
+                this.keyDownHandler(this.editor, event);
+                break;
+        }
+    }
+
+    getContentModelConfig() {
+        return {
+            domToModelOption: {
+                additionalFormatParsers: {
+                    image: [this.editingFormatParser],
+                },
+            },
+            modelToDomOption: {
+                additionalFormatAppliers: {
+                    image: [this.editingFormatApplier],
+                },
+            },
+        };
+    }
+
+    private mouseUpHandler(editor: IEditor, event: MouseUpEvent) {
+        const selection = editor.getDOMSelection();
+
+        if (
+            (selection &&
+                selection.type == 'image' &&
+                event.rawEvent.button == LEFT_MOUSE_BUTTON) ||
+            this.isEditing
+        ) {
+            this.selectionChangeHandler(editor, selection);
+        }
+    }
+
+    private keyDownHandler(editor: IEditor, event: KeyDownEvent) {
+        if (this.isEditing) {
+            const selection = editor.getDOMSelection();
+            if (!isModifierKey(event.rawEvent)) {
+                this.selectionChangeHandler(editor, selection);
+            } else if (selection?.type == 'image') {
+                this.formatImageWithContentModel(
```

Comment:
what is this for?

Reply:
Is this for applying change when moving cursor?

Reply:
If a modifier key is not pressed, the cursor will change the position, so we consider as selection change. If a modifier key is a pressed, we apply the change and leave the selection in the image.

Reply:
add comment to explain this.